i18n/en/docusaurus-plugin-content-docs/current/about/understanding/naming.md

```diff
@@ -4,59 +4,60 @@ sidebar_position: 4
 
 # Naming
 
-## Problem
+Different developers have different experiences and contexts, which can lead to misunderstandings on the team when the same entities [are called differently][disc-src]. For example:
```

Comment:
issue: in the original version, there was an abundance of links, I find them quite distracting (and also duplicated)

suggestion: remove the links from the text altogether, they're not adding a lot of meaning here
```suggestion
Different developers have different experiences and contexts, which can lead to misunderstandings on the team when the same entities are called differently. For example:
```

Reply:
Sure.